packages/vscode-extension/src/commonlib/common/constant.ts

```diff
@@ -17,3 +17,7 @@ export const envDefaultJsonFile = "env.default.json";
 export const stateDefaultJsonFile = "state.dev.json";
 
 export const m365CacheName = "appStudio";
+
+export const extensionID = "TeamsDevApp.ms-teams-vscode-extension";
+export const authComplete = "auth-complete";
```

Comment:
Since this path is only available in codespaces, how about changing to `codespacesAuthComplete` for better readability and maintenance?